arrows/klv/klv_data_format.h

```diff
@@ -86,9 +86,17 @@ class KWIVER_ALGO_KLV_EXPORT klv_data_format
   std::string
   description() const;
 
-  /// Optionally the checksum format for this data format.
+  /// Optionally the checksum format for the packet key and length only.
```

Comment:
This is minor but repeating issue: either remove the period, or turn it into a proper sentence.